js/actions/windowActions.js

```diff
@@ -638,15 +638,15 @@ const windowActions = {
    * @param {Object} frameToSkip - Properties of the frame to keep audio
    */
   muteAllAudioExcept: function (frameToSkip) {
-    let framePropsList = windowStore.getState().get('frames')
+    /* let framePropsList = windowStore.getState().get('frames')
```

Comment:
this is still necessary, but I need to refactor it, because if you require `windowStore` in appAction (you are requiring it, because I am are calling windowActions in appStore)  you are getting errors. I will move this logic from windowActions into reducer, so that we don't need store here.